script/lib/transpile-per-package-transpilation-paths.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const CompileCache = require('../../src/compile-cache')
+const fs = require('fs')
+const glob = require('glob')
+const path = require('path')
+
+const CONFIG = require('../config')
+
+module.exports = function () {
+  console.log(`Transpiling per-package transpilation paths in ${CONFIG.intermediateAppPath}`)
```

Comment:
What do you think about rewording this as `Transpiling packages with a custom transpiler`?